semantic/src/themes/universe/collections/form.variables

```diff
@@ -26,6 +26,8 @@
 ---------------------*/
 @labelFontWeight: 100;
 @labelMargin: 0 0 8px 0;
+@labelColor: #212B36;
```

Comment:
can this become a variable?

Reply:
hmm so I think this color is actually really just 1 off. It's never used in our branding or any of our other projects so I don't think it will be used again aside from this